src/components/card/card.scss

```diff
@@ -110,6 +109,8 @@
 
 .thumbnail-wrapper {
   @apply flex;
+  border-start-start-radius: var(--calcite-border-radius-base);
+  border-start-end-radius: var(--calcite-border-radius-base);
```

Comment:
This is the obvious solution but I think it breaks down in a case like this: https://codepen.io/mjuniper/pen/qBoNJpv?editors=1100.

Reply:
yeah. It does :\

Reply:
We might just have to leave it up to the user to apply a radius on their slotted image. Thoughts @jcfranco ?

Reply:
This case makes it kinda gnarly and I have not been able to find a solution yet.

Reply:
I thought maybe clip-path might help us here but I have not been able to make that work.

Reply:
We might just have to round all corners

Reply:
I'll defer to @macandcheese on this one.